src/Tasks/Microsoft.NET.Build.Tasks/EmbedAppNameInHostUtil.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Text;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    /// <summary>
+    /// Embeds the App Name into the AppHost.exe
+    /// </summary>
+    public static class EmbedAppNameInHostUtil
+    {
+        private static string _placeHolder = "c3ab8ff13720e8ad9047dd39466b3c8974e592c2fa383d4a3960714caef0c4f2"; //hash value embedded in default apphost executable
+        private static byte[] _bytesToSearch = Encoding.UTF8.GetBytes(_placeHolder);
```

Comment:
nit: pre-existing, but this can be readonly.

Reply:
marked Readonly